packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -660,7 +660,16 @@ class _UiKitViewState extends State<UiKitView> {
   }
 
   void _onFocusChange(bool isFocused) {
-    // TODO(hellohuanlin): send 'TextInput.setPlatformViewClient' channel message to engine after the engine is updated to handle this message.
+    if (!isFocused) {
```

Comment:
nits:
Maybe this would be more readable if we check if (isFocused) first?
```dart
if (isFocused) {
  SystemChannels.textInput.invokeMethod<void>(
        'TextInput.setPlatformViewClient',
        <String, dynamic>{'platformViewId': _controller!.id},

}
```

Or we can escape early instead of if-else

```dart
if (!isfocused) {
  return;
}
 SystemChannels.textInput.invokeMethod<void>(
        'TextInput.setPlatformViewClient',
        <String, dynamic>{'platformViewId': _controller!.id},